grails-app/assets/javascripts/person-form-controller.js

```diff
@@ -0,0 +1,27 @@
+function PersonFormController() {
+    var _this = this;
+    _this.reference = $(".js-customer-create");
+
+    _this.init = function() {
+        var postalCodeUtils = new PostalCodeUtils();
+        postalCodeUtils.init();
+
+        _this.buildCustomerCreateEmail();
+    }
+
+    _this.buildCustomerCreateEmail = function() {
+        if (_this.isCustomerCreate()) {
+            var email = _this.reference.find(".js-email")
```

Comment:
```suggestion
            var email = _this.reference.find(".js-email");
```
Confere o ponto e virgula